components/Settings.js

```diff
@@ -116,7 +115,7 @@ const Settings = (props) => {
             ))}
           </Picker>
         </View>
-        { regionKey && <SettingsItem term="settings/push_locations" onPress={showPushDialog} disabled={!canUpload} /> }
+        { regionKey && <SettingsItem term="settings/location_data" onPress={locationData} /> }
```

Comment:
Should we feature flag between the normal behaviour and the new screen? That way we can merge it faster and start getting feedback from rob.

Reply:
Oooh ooh. We can also get the translations going.

Reply:
The more I think about it, the keener I am to feature flag this. We could ship without it and then light it up after. We could even use the language/translations as part of the feature flag.

Reply:
Expo constants for feature flag? Or what would you prefer?

Reply:
Either expo constant or even `__DEV__` for now. We can get fancier if needed.

Reply:
Used __DEV__ for now